### DISCUSSION TITLE
Does not recognize the name of the docker service in the env

### DISCUSSION THREAD

**Discussion by MrMagloire** [prodrigestivill/docker-postgres-backup-local#105] [Q&A] [answered]
Hello, thank you for all you are making for all with this docker image.
But I have some problem with my docker-compose environnement variable, it Does not recognize the name of the docker service in the env

          auth-db:
            image: postgres:12.0-alpine
            restart: always
            container_name: evoire-auth-db
            networks:
              - evoire-auth-net
            volumes:
              - auth_user_data:/var/lib/postgresql/data/
            env_file:
              - ./.env
  
          auth-pgbackups:
            container_name: Backup
            image: prodrigestivill/postgres-backup-local
            restart: always
            volumes:
              - ./backup:/backups
            links:
              - auth-db
            depends_on:
              - auth-db
            environment:
              - POSTGRES_HOST=auth-db
              - POSTGRES_DB=${POSTGRES_DB} 
              - POSTGRES_USER=${POSTGRES_USER}
              - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
              - POSTGRES_EXTRA_OPTS=-Z9 --schema=public --blobs
              - SCHEDULE=*/5 * * * *
              - BACKUP_KEEP_DAYS=7
              - BACKUP_KEEP_WEEKS=4
              - BACKUP_KEEP_MONTHS=6
              - HEALTHCHECK_PORT=81

**Comment by prodrigestivill** [comment, marked as answer]
You need to have both conainers in at least one common network.

In your example auth-pgbackups is not in evoire-auth-net.
